src/data/endpoints/account.yaml

```diff
@@ -5,28 +5,26 @@ description: >
   Account endpoints provide a means of manipulating your account, including users,
   OAuth Clients and Tokens, and Events.
 endpoints:
-  /account/settings:
-    group: Settings
+  /account:
     type: resource
     description: >
-      Manage your account settings.
+        Managed account contact and billing information.
     methods:
       GET:
         response: account
         description: >
           Returns your account settings.
         examples:
           curl: |
-            curl https://$api_root/$version/account/settings
-          python: |
-            my_settings = client.account.get_settings()
+            curl -h "Authorization: Bearer $TOKEN" \
+                https://$api_root/$version/account
       PUT:
         description: >
           Edits your account settings.
         examples:
           curl: |
-            curl -H "Content-Type: application/json" \
-                -H "Authorization: Bearer $TOKEN" \
+            curl -h "Content-Type: application/json" \
```

Comment:
why did these get changed to `-h` from `-H` when the man for curl says `-H` is correct?

Reply:
😞  I think I hit `u` instead of `y` one time and didn't notice